screenpipe-core/src/operator/platforms/windows.rs

```diff
@@ -87,140 +301,272 @@ impl Debug for WindowsUIElement {
 
 impl UIElementImpl for WindowsUIElement {
     fn object_id(&self) -> usize {
-        0
+        // similar to macos :D
+        let stable_id = format!("{:?}",
```

Comment:
on macos we use hash of property becuse macos does not provide id but if windows has id for element that would be better idk